packages/@o3r/components/src/tools/component-replacement/c11n.directive.ts

```diff
@@ -95,8 +95,6 @@ export class C11nDirective<
     if (changes.component && changes.component.currentValue) {
 
       if (this.componentRef) {
-        this.componentSubscriptions.forEach((s) => s.unsubscribe());
```

Comment:
this one might still be needed to cleanup the subscriptions when changing the `componentRef`

Reply:
fallback to ensure no regression on this one